### DISCUSSION TITLE
strange server problem

### DISCUSSION THREAD

**Discussion by Bhrighde** [jamulussoftware/jamulus#2622] [Technical Support & Help] [answered]
Hi everyone.  We've encountered a strange issue that just started over the last week or so.  We  set up one computer as our server that has worked fine for over a year.  The server could be found using  Jamulus by any of the computers that were part of the LAN that the server is part of, as well as musicians accessing it from locations literally around  the world. 
 
However, as of last Sunday, April 24, the LAN computers can no longer access the server, while outside users are still able to  access it.  We have tested it several times since then and tried to troubleshoot the problem, but it persists.

We've checked with the ISP, and everything is fine.  Our port forwarding is good, otherwise our external folks wouldn't be able to access it.

Does anyone have any idea what could cause this?  We're at our wits end.

**Comment by pgScorpio** [comment]
Well, apparently something changed  on Sunday, April 24 ;=)

Most likely firewall settings changed, either on the computer that runs the server or on the router (Maybe you got an automatic firmware update on the modem/router, or the modem/router was reset by the ISP due to a network upgrade?)

If that's not the case it's hard to tell without more information like server configuration (commandline?), which router is used, used OS'es, what URL/IP do you use to connect to the server, etc...

**Comment by Bhrighde** [comment, marked as answer]
I finally did a deep dive into the virus protection software on the server (it had just been updated a week ago). I found it had the local network pegged as 'untrustworthy'! So I changed that, and it all works fine!
